staging/src/k8s.io/kubectl/pkg/cmd/create/create_namespace.go

```diff
@@ -17,8 +17,21 @@ limitations under the License.
 package create
 
 import (
+	"context"
+	"io"
+
 	"github.com/spf13/cobra"
 
+	"k8s.io/apimachinery/pkg/api/meta"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	kruntime "k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/cli-runtime/pkg/resource"
+	"k8s.io/client-go/dynamic"
```

Comment:
What about using the typed client: https://github.com/kubernetes/client-go/tree/master/kubernetes/typed/core/v1